File: CodeForces/401-500/405/A/405A.py
# Problem 405A: Gravity Flip
# https://codeforces.com/contest/405/problem/A
# Executed with: Python 3.11.2
# Linux version: Linux kernel 6.1.0-10-amd64
# Submitted on: September 27th, 2024
# Codeforces language used: PyPy 3.10 (7.3.15, 64bit)

rows = int(input())
heights = sorted(list(map(int, input().split())))
print(" ".join(str(num) for num in heights))

File: CodeForces/401-500/405/A/405A.rs
// Problem 405A: Gravity Flip
// https://codeforces.com/contest/405/problem/A
// Compiled with: rustc 1.85.1 (4eb161250 2025-03-15) (built from a source tarball)
// Linux version: Linux kernel 6.17.0-5-generic
// Submitted on: October 26th, 2025
// Codeforces language used: Rust 1.89.0 (2024)

fn main() {
    // n
    let _n: i32 = std::io::stdin().lines().next().unwrap().unwrap().trim().parse().unwrap();

    // a1...an
    let mut nums: Vec<i32> = std::io::stdin().lines().next().unwrap().unwrap().split_whitespace().map(|x| x.parse::<i32>().unwrap()).collect();
    nums.sort_by_key(|x| *x);

    for val in nums {
        print!("{}", val);
        print!(" ");
    }
    print!("\n");
}
